README.md

```diff
@@ -160,17 +160,17 @@ $ ballerina run book_search/
 2. Next, run the three instances of the book store service. Here you have to enter the service port number in each service instance. You can pass the port number as parameter `Bport=<Port Number>`.
 ``` bash
 // 1st instance with port number 9011
-$ ballerina run book_store_backed -e port=9011
+$ ballerina run -e port=9011 book_store_backed 
```

Comment:
should this all be "book_store_backe**n**d"?

Reply:
Seems like it 🤔